pkgs/applications/misc/synergy/default.nix

```diff
@@ -24,7 +24,7 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = lib.optional (!withGUI) "-DSYNERGY_BUILD_LEGACY_GUI=OFF";
```

Comment:
looks like this has changed:
```
CMake Warning at src/lib/arch/CMakeLists.txt:46 (find_package):
  By not providing "FindQt5.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "Qt5", but
  CMake did not find one.

  Could not find a package configuration file provided by "Qt5" with any of
  the following names:

    Qt5Config.cmake
    qt5-config.cmake
```

Reply:
```
https://github.com/NixOS/nixpkgs/pull/144267

1 package failed to build:
synergyWithoutGUI

2 packages built:
quicksynergy synergy
```

Reply:
i'll see what I can do here, i'm amazed at everything you all do! Thank you for putting up with me!

Reply:
I added a question on the nativeBuildInputs section, does adding pkg-config there break this? Net sure where or how this is failing.